src/pages/SchoolsGridPage.tsx

```diff
@@ -59,12 +58,13 @@ const LoadingCard = styled(Card)`
   ${createPlaceholderStyles()}
   height: 200px;
   box-shadow: none;
+  margin-bottom: 3em;
   &::after {
     background: #eee;
   }
 `;
 
-const Results: FC<any> = ({ schools, error, page, count, onPageChange }) => {
+const Results: FC<any> = ({ schools, error }) => {
   if (error) return <ErrorInfo />;
```

Comment:
This file is quite long, maybe we can move Results to a separate file? And all visual components could also be in the separate file